tests/linkname_test.go

```diff
@@ -24,3 +25,12 @@ func TestLinknames(t *testing.T) {
 		t.Fatalf("Callink linknamed functions returned a diff (-want,+got):\n%s", diff)
 	}
 }
+
+func TestLinknameMethods(t *testing.T) {
+	defer func() {
+		if err := recover(); err != nil {
+			t.Fatalf("method.TestLinkname() paniced: %s", err)
+		}
+	}()
+	method.TestLinkname()
+}
```

Comment:
Very nice to see a comprehensive set of tests, thank you!

One suggestion though: the way this test is currently written, it would be nearly impossible to determine where the panic came from, especially considering that the panic message is  just the bare "got" value with no context. I would suggest reorganizing the tests like so:

```go
// In .../method/method.go:

func TestString(t *testing.T) {
	var i _string
	string_Set(&i, "hello world")
	got := string_Get(i)
	want := "hello world"
	if  got != want{
		t.Fatalf("Got: string_Get(i) = %q. Want: %q.",  got, want)
	}
}

// In tests/linkname_test.go:
func TestLinknameMethods(t *testing.T) {
	t.Run("struct", method.TestStruct)
	// and so on...
	t.Run("string", method.TestString)
}
```

In this example we make use of sub-tests, which will allow the Go test framework to tell us which case exactly broke, and more detailed error message would help diagnosing the problem right away. Does that make sense?

Reply:
use testing.T
but not use t.Run for catch panic error